Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -1611,6 +1608,20 @@ private void ActAccessibility(ActAccessibilityTesting act)
                     axeResult = axeBuilder.Analyze();
                 }
 
+                string path = String.Empty;
+
+                if(WorkSpace.Instance.Solution.LoggerConfigurations.CalculatedLoggerFolder != null)
+                {
+                    string folderPath = Path.Combine(WorkSpace.Instance.Solution.LoggerConfigurations.CalculatedLoggerFolder, @"AccessibilityReport\");
```

Comment:
User Path.DirectorySeparatorChar here as well instead of \

Reply:
Done